api/hubble/history_trades.mdx

```diff
@@ -0,0 +1,28 @@
+| batch_id | string | String representation of the run id for a given DAG in Airflow. Takes the form of "scheduled__<batch_end_date>-<dag_alias>". Batch ids are unique to the batch and help with monitoring and rerun capabilities |
+| batch_run_date | datetime | The start date for the batch interval. When taken with the date in the batch_id, the date represents the interval of ledgers processed. The batch run date can be seen as a proxy of closed_at for a ledger. |
+| batch_insert_ts | timestamp | The timestamp in UTC when a batch of records was inserted into the database. This field can help identify if a batch executed in real time or as part of a backfill. The timestamp should not be used during ad hoc analysis and is useful for data engineering purposes. |
+| selling_liquidity_pool_id | string | The unique identifier for a liquidity pool if the trade was executed against a liquidity pool instead of the orderbook |
+| liquidity_pool_fee | int64 | The percentage fee (in basis points) of the total fee collected by the liquidity pool for executing the trade.<br />The fee is pooled and distributed back to liquidity pool shareholders to incentivize users to stake money in the pool.<br /><br /><table> <thead> <tr> <th>Default Values</th> </tr> </thead> <tbody> <tr> <td>30</td> </tr> </tbody> </table><br /><br />#### Notes:<br />Liquidity pool fees can only change with protocol changes to the network itself |
+| trade_type | int64 | Indicates whether the trade was executed against the orderbook (decentralized exchange) or liquidity pool.<br /><br /><table> <thead> <tr> <th>Code</th> <th>Value</th> </tr> </thead> <tbody> <tr> <td>1</td> <td>Descentralized Exchange Trade</td> </tr> <tr> <td>2</td> <td>Liquidity Pool Trade</td> </tr> </tbody> </table> |
+| rounding_slippage | int64 | Applies to liquidity pool trades only. With fractional amounts of an asset traded, the network must round a fraction to the nearest whole number. This can cause the trade to "slip" price by a percentage compared with the original offer. Rounding slippage reports the percentage that dust trades slip before executing.<br /><br />#### Notes:<br />Defaults to 1<br />Rounding Slippage is always unprofitable for the trader and is not a valid way to try and extract more value from the network. |
+| seller_is_exact | bool | Indicates whether the buying or selling party trade was impacted by rounding slippage. If true, the buyer was impacted. If false, the seller was impacted |
+| selling_asset_id | int64 | Unique identifier for the asset code, type and issuer combination. This is not a primary key on the table<br /><br />- Natural Key<br />- Required |
+| buying_asset_id | int64 | Unique identifier for the asset code, type and issuer combination. This is not a primary key on the table<br /><br />- Natural Key<br />- Required |
```

Comment:
Might be worthwhile to point out that joining on asset_id is more efficient since it's an integer. That's the only reason why we added these fields onto the table